cli/graph-nodes.ts

```diff
@@ -40,8 +40,16 @@ export const graphNodeInfoByNetwork: GraphNodeInfoByNetwork = {
     index: "https://thegraph.coredao.org/graphql",
     ipfs: "https://thegraph.coredao.org/ipfs/",
   },
+  // Zksync era deployment was done via subgraph studio
+  zksyncera: {
+    queryBase: "https://api.studio.thegraph.com/query/35843/request-payment-zksyncera/version/latest",
+    deploy: "",
+    index: "",
+    ipfs: ""
+  }
```

Comment:
I could not find these urls for this subgraph in the [Studio page](https://thegraph.com/studio/subgraph/request-payment-zksyncera/)
@benjlevesque is that ok?

Reply:
you should only reference chains that aren't supported by the studio. You can remove zksyncera from this list